Services/FileService.cs

```diff
@@ -25,35 +26,38 @@ public void BackupUserSubmission(string path)
 			File.WriteAllText(backupLocation, content);
 		}
 
-		public void CreateOrOverwriteUserSubmission(string username, string submission)
+		public void CreateOrOverwriteUserSubmission(string antName, string userName, string submission)
 		{
-			if (UserSubmissionExists(username))
+			string fileName = $"{userName}_{antName}"; 
+			if (UserSubmissionExists(fileName))
 			{
-				BackupUserSubmission(GetUserSubmissionPath(username));
+				BackupUserSubmission(GetUserSubmissionPath(fileName));
 			}
 
-			using (FileStream fs = System.IO.File.Create(GetUserSubmissionPath(username)))
+			using (FileStream fs = File.Create(GetUserSubmissionPath(fileName)))
 			{
 				byte[] content = new UTF8Encoding(true).GetBytes(submission);
 
 				fs.Write(content, 0, content.Length);
 			}
 		}
 
-		public void DeleteUserSubmission(string username)
+		public void DeleteUserSubmission(string userName, string antName)
```

Comment:
The params here are backwards based on what you're passing in through the controller